StoreApp/Models/Store/CreateProductViewModel.cs

```diff
@@ -9,8 +9,11 @@ public class CreateProductViewModel
         [Required]
         public string Name { get; set; }
         [Required]
+        [RegularExpression(@"^\d*(,)?(\d?\d?)?$", ErrorMessage = "Invalid characters")]
+        [Range(0.01, 999999999999, ErrorMessage = "Invalid value")]
         public decimal Price { get; set; }
```

Comment:
You have set up the range of values here but you check the value before (**productDTO.Price.Equals(0)**)

Reply:
change to Quantity